src/components/ContinuationIn/ExtraproRegistration.vue

```diff
@@ -428,11 +389,10 @@
 <script lang="ts">
 import { Component, Emit, Mixins, Watch } from 'vue-property-decorator'
 import { Action, Getter } from 'pinia-class'
-import { StatusCodes } from 'http-status-codes'
 import { mask } from 'vue-the-mask'
 import { useStore } from '@/store/store'
 import { DateMixin, DocumentMixin } from '@/mixins'
```

Comment:
Do you still need DocumentMixin in this component?

Reply:
What about PdfPageSize?

Reply:
Removed unused codes